### PR TITLE
Update ci badge

### PR DESCRIPTION
**Pull request by travissanderson-wf**

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by danielway-wk — APPROVED** [review]
+10

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# RTree [![Build Status](https://travis-ci.org/Workiva/r_tree.svg)](https://travis-ci.org/Workiva/r_tree)
```
Comment (brettmeyer-wk) [inline README.md:1]: Even though this is a readme only change, I prefer every PR to have a description with an explanation of why these change are being done, however brief.

**Review by brettmeyer-wk — APPROVED** [review]

**Review by kerbyseeley-wf — APPROVED** [review]

**Comment by travissanderson-wf** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM
